doc/flame/components.md

```diff
@@ -340,6 +340,12 @@ void onDragUpdate(DragUpdateInfo info) {
 
 ### PositionType
 
+```{note}
+If you are using the `CameraComponent` you should not use `PositionType`, but
```

Comment:
Is this related to this new component?

Reply:
It is not, I just saw that the docs missed it since I added this component to the section underneath.

Reply:
Got ya!